transaction_integration_test.go

```diff
@@ -435,6 +439,9 @@ func TestAllTransactionFields(t *testing.T) {
 	if tx2.ShippingAddress.PostalCode != tx.ShippingAddress.PostalCode {
 		t.Fatalf("expected ShippingAddress.PostalCode to be equal, but %s was not %s", tx2.ShippingAddress.PostalCode, tx.ShippingAddress.PostalCode)
 	}
+	if tx2.TaxAmount.Cmp(tx.TaxAmount) != 0 {
+		t.Fatalf("expected TaxAmount to be equal, but %s was not %s", tx2.TaxAmount, tx.TaxAmount)
+	}
```

Comment:
Could we add an expectation for the value of the `TaxExempt` field in this test?